### ISSUE TITLE
Convert model Properties to standard units? 

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/my-solar-system#252]
Related to https://github.com/phetsims/my-solar-system/issues/229 ...

@AgustinVallejo had the nice idea of changing how standard vs non-standard units are handled.  The model (e.g. Body) would use standard units for its Properties.  And the conversion to/from non-standard units would be handled where it's needed -- presumably in Engine -- so that we don't run into problems with excessively large or small numbers.

Mass could still be specified as N in N x 10<sup>28</sup> kg.

Ideally we would also specify BodyInfo in standard units.

This would solve several problems for PhET-iO:
* We would not need to devise a way to allow PhET-iO clients to enter Property values in standard units.
* We would eliminate the confusion of having Properties for both standard and non-standard units in the model.
* We would no longer need the MappedProperty instances under `valuesPanel`. 

@AgustinVallejo will investigate the feasibility of this change.

**Comment by pixelzoom** [comment]
I'm going to label this issue as high priority, because it seem to be (currently) the most important issue for PhET-iO instrumentation.

**Comment by AgustinVallejo** [comment]
In the exploration phase I began changing the position property to the standard units. This was the simplest change as it only decreases by a factor of 100, i.e. if the starting position was 125 it's now 1.25. The current state of the sim is promising as it seems the gravitational system is still intact. 

Below is a patch of the changes, @pixelzoom let me know how you think we should handle these changes: Should I proceed with an incomplete commit to origin? Should we work on a branch while we make sure not to have broken anything? (we'd have to create a branch per repo) or should I work locally until I'm sure this works?

<details>
<summary>Click to expand/collapse</summary>

```ts
Subject: [PATCH] Changing back names to GravityZoom, see https://github.com/phetsims/my-solar-system/issues/235
---
Index: my-solar-system/js/common/view/ValuesColumnNode.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/my-solar-system/js/common/view/ValuesColumnNode.ts b/my-solar-system/js/common/view/ValuesColumnNode.ts
--- a/my-solar-system/js/common/view/ValuesColumnNode.ts	(revision 1d7455af46b4819f625f0eba5198db0be6e6e513)
+++ b/my-solar-system/js/common/view/ValuesColumnNode.ts	(date 1697597626359)
@@ -142,8 +142,8 @@
         phetioValueType: NumberIO,
         reentrant: true,
         bidirectional: true,
-        map: position => position.x * SolarSystemCommonConstants.POSITION_MULTIPLIER,
-        inverseMap: ( x: number ) => new Vector2( x / SolarSystemCommonConstants.POSITION_MULTIPLIER, body.positionProperty.value.y ),
+        map: position => position.x,
+        inverseMap: ( x: number ) => new Vector2( x, body.positionProperty.value.y ),
         tandem: parentTandem.createTandem( `positionX${body.index}MappedProperty` )
       } );
 
@@ -168,8 +168,8 @@
         phetioValueType: NumberIO,
         reentrant: true,
         bidirectional: true,
-        map: position => position.y * SolarSystemCommonConstants.POSITION_MULTIPLIER,
-        inverseMap: ( y: number ) => new Vector2( body.positionProperty.value.x, y / SolarSystemCommonConstants.POSITION_MULTIPLIER ),
+        map: position => position.y,
+        inverseMap: ( y: number ) => new Vector2( body.positionProperty.value.x, y ),
         tandem: parentTandem.createTandem( `positionY${body.index}MappedProperty` )
       } );
 
Index: my-solar-system/js/intro/model/IntroModel.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/my-solar-system/js/intro/model/IntroModel.ts b/my-solar-system/js/intro/model/IntroModel.ts
--- a/my-solar-system/js/intro/model/IntroModel.ts	(revision 1d7455af46b4819f625f0eba5198db0be6e6e513)
+++ b/my-solar-system/js/intro/model/IntroModel.ts	(date 1697596683620)
@@ -22,7 +22,7 @@
       // MySolarSystemModelOptions
       defaultBodyInfo: [
         new BodyInfo( { isActive: true, mass: 250, position: new Vector2( 0, 0 ), velocity: new Vector2( 0, -11.1 ) } ),
-        new BodyInfo( { isActive: true, mass: 25, position: new Vector2( 200, 0 ), velocity: new Vector2( 0, 111 ) } )
+        new BodyInfo( { isActive: true, mass: 25, position: new Vector2( 2.00, 0 ), velocity: new Vector2( 0, 111 ) } )
       ],
       tandem: tandem
     } );
Index: my-solar-system/js/common/model/NumericalEngine.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/my-solar-system/js/common/model/NumericalEngine.ts b/my-solar-system/js/common/model/NumericalEngine.ts
--- a/my-solar-system/js/common/model/NumericalEngine.ts	(revision 1d7455af46b4819f625f0eba5198db0be6e6e513)
+++ b/my-solar-system/js/common/model/NumericalEngine.ts	(date 1697597832358)
@@ -14,7 +14,7 @@
 import Vector2 from '../../../../dot/js/Vector2.js';
 import Body from '../../../../solar-system-common/js/model/Body.js';
 import Engine from '../../../../solar-system-common/js/model/Engine.js';
-import { G } from '../../../../solar-system-common/js/SolarSystemCommonConstants.js';
+import SolarSystemCommonConstants, { G } from '../../../../solar-system-common/js/SolarSystemCommonConstants.js';
 
 // Used when we want to avoid modifying a Vector (position, velocity, acceleration),
 // and want to avoid allocating a new Vector copy.
@@ -86,8 +86,8 @@
     dt /= iterationCount;
 
     const masses = this.bodies.map( body => body.massProperty.value );
-    const positions = this.bodies.map( body => body.positionProperty.value.copy() );
-    const velocities = this.bodies.map( body => body.velocityProperty.value.copy() );
+    const positions = this.bodies.map( body => body.positionProperty.value.copy().dividedScalar( SolarSystemCommonConstants.POSITION_MULTIPLIER ) );
+    const velocities = this.bodies.map( body => body.velocityProperty.value.copy().dividedScalar( SolarSystemCommonConstants.POSITION_MULTIPLIER ) );
     const accelerations = this.bodies.map( body => body.accelerationProperty.value.copy() );
     const forces = this.bodies.map( body => body.forceProperty.value.copy() );
 
@@ -185,14 +185,14 @@
       // Currently, on SolarSystemCommonModel, it is set to false except on the last iteration of the model.run() method.
 
       if ( updateProperties ) {
-        body.positionProperty.value = positions[ i ];
-        body.velocityProperty.value = velocities[ i ];
+        body.positionProperty.value = positions[ i ].multiplyScalar( SolarSystemCommonConstants.POSITION_MULTIPLIER );
+        body.velocityProperty.value = velocities[ i ].multiplyScalar( SolarSystemCommonConstants.POSITION_MULTIPLIER );
         body.accelerationProperty.value = accelerations[ i ];
         body.forceProperty.value = forces[ i ];
       }
       else {
-        body.positionProperty.value.set( positions[ i ] );
-        body.velocityProperty.value.set( velocities[ i ] );
+        body.positionProperty.value.set( positions[ i ].multiplyScalar( SolarSystemCommonConstants.POSITION_MULTIPLIER ) );
+        body.velocityProperty.value.set( velocities[ i ].multiplyScalar( SolarSystemCommonConstants.POSITION_MULTIPLIER ) );
         body.accelerationProperty.value.set( accelerations[ i ] );
         body.forceProperty.value.set( forces[ i ] );
       }
Index: solar-system-common/js/model/Body.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/solar-system-common/js/model/Body.ts b/solar-system-common/js/model/Body.ts
--- a/solar-system-common/js/model/Body.ts	(revision 8bc3891932b141a5a21ad0ba62170d6c0b24994b)
+++ b/solar-system-common/js/model/Body.ts	(date 1697597544590)
@@ -96,7 +96,7 @@
       phetioReadOnly: true
     } );
 
-    this.velocityProperty = new Vector2Property( bodyInfo.velocity, {
+    this.velocityProperty = new Vector2Property( bodyInfo.velocity.multiplyScalar( 0.01 ), {
       //TODO https://github.com/phetsims/my-solar-system/issues/244 units
       tandem: tandem.createTandem( 'velocityProperty' ),
       phetioReadOnly: true
@@ -234,8 +234,8 @@
   }
 
   public static massToRadius( mass: number ): number {
-    const minRadius = 3;
-    return Math.max( minRadius, 2.3 * Math.pow( mass, 1 / 3 ) );
+    const minRadius = 0.03;
+    return Math.max( minRadius, 0.023 * Math.pow( mass, 1 / 3 ) );
   }
 
   /**
Index: keplers-laws/js/common/model/KeplersLawsModel.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/keplers-laws/js/common/model/KeplersLawsModel.ts b/keplers-laws/js/common/model/KeplersLawsModel.ts
--- a/keplers-laws/js/common/model/KeplersLawsModel.ts	(revision 8d6b4b00b9030abdc34345300d4b5c2ba5f6bf1d)
+++ b/keplers-laws/js/common/model/KeplersLawsModel.ts	(date 1697596663201)
@@ -106,7 +106,7 @@
       modelToViewTime: 1 / 12.7,
       defaultBodyInfo: [
         new BodyInfo( { isActive: true, mass: 200, position: new Vector2( 0, 0 ), velocity: new Vector2( 0, 0 ), tandemName: 'sun' } ),
-        new BodyInfo( { isActive: true, mass: 50, position: new Vector2( 200, 0 ), velocity: new Vector2( 0, 81.6 ), tandemName: 'planet' } )
+        new BodyInfo( { isActive: true, mass: 50, position: new Vector2( 2.00, 0 ), velocity: new Vector2( 0, 81.6 ), tandemName: 'planet' } )
       ]
     }, providedOptions );
     super( options );
Index: my-solar-system/js/lab/model/LabModel.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/my-solar-system/js/lab/model/LabModel.ts b/my-solar-system/js/lab/model/LabModel.ts
--- a/my-solar-system/js/lab/model/LabModel.ts	(revision 1d7455af46b4819f625f0eba5198db0be6e6e513)
+++ b/my-solar-system/js/lab/model/LabModel.ts	(date 1697596892488)
@@ -27,9 +27,9 @@
       // MySolarSystemModelOptions
       defaultBodyInfo: [
         new BodyInfo( { isActive: true, mass: 250, position: new Vector2( 0, 0 ), velocity: new Vector2( 0, -11.1 ) } ),
-        new BodyInfo( { isActive: true, mass: 25, position: new Vector2( 200, 0 ), velocity: new Vector2( 0, 111 ) } ),
-        new BodyInfo( { isActive: false, mass: 0.1, position: new Vector2( 100, 0 ), velocity: new Vector2( 0, 150 ) } ),
-        new BodyInfo( { isActive: false, mass: 0.1, position: new Vector2( -100, -100 ), velocity: new Vector2( 120, 0 ) } )
+        new BodyInfo( { isActive: true, mass: 25, position: new Vector2( 2.00, 0 ), velocity: new Vector2( 0, 111 ) } ),
+        new BodyInfo( { isActive: false, mass: 0.1, position: new Vector2( 1.00, 0 ), velocity: new Vector2( 0, 150 ) } ),
+        new BodyInfo( { isActive: false, mass: 0.1, position: new Vector2( -1.00, -1.00 ), velocity: new Vector2( 120, 0 ) } )
       ],
       isLab: true,
       tandem: tandem
@@ -104,69 +104,69 @@
   private initializeModeMap(): void {
     this.orbitalSystemMap.set( OrbitalSystem.SUN_PLANET, [
       new BodyInfo( { isActive: true, mass: 250, position: new Vector2( 0, 0 ), velocity: new Vector2( 0, -11.1 ) } ),
-      new BodyInfo( { isActive: true, mass: 25, position: new Vector2( 200, 0 ), velocity: new Vector2( 0, 111 ) } )
+      new BodyInfo( { isActive: true, mass: 25, position: new Vector2( 2.00, 0 ), velocity: new Vector2( 0, 111 ) } )
     ] );
     this.orbitalSystemMap.set( OrbitalSystem.SUN_PLANET_MOON, [
       new BodyInfo( { isActive: true, mass: 200, position: new Vector2( 0, 0 ), velocity: new Vector2( 0, 0 ) } ),
-      new BodyInfo( { isActive: true, mass: 10, position: new Vector2( 160, 0 ), velocity: new Vector2( 0, 120 ) } ),
-      new BodyInfo( { isActive: true, mass: 0.000001, position: new Vector2( 140, 0 ), velocity: new Vector2( 0, 53 ) } )
+      new BodyInfo( { isActive: true, mass: 10, position: new Vector2( 1.60, 0 ), velocity: new Vector2( 0, 120 ) } ),
+      new BodyInfo( { isActive: true, mass: 0.000001, position: new Vector2( 1.40, 0 ), velocity: new Vector2( 0, 53 ) } )
     ] );
     this.orbitalSystemMap.set( OrbitalSystem.SUN_PLANET_COMET, [
       new BodyInfo( { isActive: true, mass: 200, position: new Vector2( 0, 0 ), velocity: new Vector2( 0, 0 ) } ),
-      new BodyInfo( { isActive: true, mass: 1, position: new Vector2( 150, 0 ), velocity: new Vector2( 0, 120 ) } ),
-      new BodyInfo( { isActive: true, mass: 0.000001, position: new Vector2( -220, 130 ), velocity: new Vector2( -20, -35 ) } )
+      new BodyInfo( { isActive: true, mass: 1, position: new Vector2( 1.50, 0 ), velocity: new Vector2( 0, 120 ) } ),
+      new BodyInfo( { isActive: true, mass: 0.000001, position: new Vector2( -2.20, 1.30 ), velocity: new Vector2( -20, -35 ) } )
     ] );
     this.orbitalSystemMap.set( OrbitalSystem.TROJAN_ASTEROIDS, [
       new BodyInfo( { isActive: true, mass: 200, position: new Vector2( 0, 0 ), velocity: new Vector2( 0, 0 ) } ),
-      new BodyInfo( { isActive: true, mass: 5, position: new Vector2( 150, 0 ), velocity: new Vector2( 0, 119 ) } ),
-      new BodyInfo( { isActive: true, mass: 0.000001, position: new Vector2( 75, -130 ), velocity: new Vector2( 103, 60 ) } ),
-      new BodyInfo( { isActive: true, mass: 0.000001, position: new Vector2( 75, 130 ), velocity: new Vector2( -103, 60 ) } )
+      new BodyInfo( { isActive: true, mass: 5, position: new Vector2( 1.50, 0 ), velocity: new Vector2( 0, 119 ) } ),
+      new BodyInfo( { isActive: true, mass: 0.000001, position: new Vector2( 0.75, -1.30 ), velocity: new Vector2( 103, 60 ) } ),
+      new BodyInfo( { isActive: true, mass: 0.000001, position: new Vector2( 0.75, 1.30 ), velocity: new Vector2( -103, 60 ) } )
     ] );
     this.orbitalSystemMap.set( OrbitalSystem.ELLIPSES, [
-      new BodyInfo( { isActive: true, mass: 250, position: new Vector2( -200, 0 ), velocity: new Vector2( 0, 0 ) } ),
-      new BodyInfo( { isActive: true, mass: 0.000001, position: new Vector2( -115, 0 ), velocity: new Vector2( 0, 151 ) } ),
-      new BodyInfo( { isActive: true, mass: 0.000001, position: new Vector2( 50, 0 ), velocity: new Vector2( 0, 60 ) } ),
-      new BodyInfo( { isActive: true, mass: 0.000001, position: new Vector2( 220, 0 ), velocity: new Vector2( 0, 37 ) } )
+      new BodyInfo( { isActive: true, mass: 250, position: new Vector2( -2.00, 0 ), velocity: new Vector2( 0, 0 ) } ),
+      new BodyInfo( { isActive: true, mass: 0.000001, position: new Vector2( -1.15, 0 ), velocity: new Vector2( 0, 151 ) } ),
+      new BodyInfo( { isActive: true, mass: 0.000001, position: new Vector2( 0.50, 0 ), velocity: new Vector2( 0, 60 ) } ),
+      new BodyInfo( { isActive: true, mass: 0.000001, position: new Vector2( 2.20, 0 ), velocity: new Vector2( 0, 37 ) } )
     ] );
     this.orbitalSystemMap.set( OrbitalSystem.HYPERBOLIC, [
-      new BodyInfo( { isActive: true, mass: 250, position: new Vector2( 0, 25 ), velocity: new Vector2( 0, 0 ) } ),
-      new BodyInfo( { isActive: true, mass: 0.000001, position: new Vector2( -250, -70 ), velocity: new Vector2( 120, 0 ) } ),
-      new BodyInfo( { isActive: true, mass: 0.000001, position: new Vector2( -250, -140 ), velocity: new Vector2( 120, 0 ) } ),
-      new BodyInfo( { isActive: true, mass: 0.000001, position: new Vector2( -250, -210 ), velocity: new Vector2( 120, 0 ) } )
+      new BodyInfo( { isActive: true, mass: 250, position: new Vector2( 0, 0.25 ), velocity: new Vector2( 0, 0 ) } ),
+      new BodyInfo( { isActive: true, mass: 0.000001, position: new Vector2( -2.50, -0.70 ), velocity: new Vector2( 120, 0 ) } ),
+      new BodyInfo( { isActive: true, mass: 0.000001, position: new Vector2( -2.50, -1.40 ), velocity: new Vector2( 120, 0 ) } ),
+      new BodyInfo( { isActive: true, mass: 0.000001, position: new Vector2( -2.50, -2.10 ), velocity: new Vector2( 120, 0 ) } )
     ] );
     this.orbitalSystemMap.set( OrbitalSystem.SLINGSHOT, [
-      new BodyInfo( { isActive: true, mass: 200, position: new Vector2( 1, 0 ), velocity: new Vector2( 0, -1 ) } ),
-      new BodyInfo( { isActive: true, mass: 10, position: new Vector2( 131, 55 ), velocity: new Vector2( -55, 115 ) } ),
-      new BodyInfo( { isActive: true, mass: 0.000001, position: new Vector2( -6, -128 ), velocity: new Vector2( 83, 0 ) } )
+      new BodyInfo( { isActive: true, mass: 200, position: new Vector2( 1.00, 0 ), velocity: new Vector2( 0, -1 ) } ),
+      new BodyInfo( { isActive: true, mass: 10, position: new Vector2( 1.31, 0.55 ), velocity: new Vector2( -55, 115 ) } ),
+      new BodyInfo( { isActive: true, mass: 0.000001, position: new Vector2( -0.06, -1.28 ), velocity: new Vector2( 83, 0 ) } )
     ] );
     this.orbitalSystemMap.set( OrbitalSystem.DOUBLE_SLINGSHOT, [
       new BodyInfo( { isActive: true, mass: 200, position: new Vector2( 0, 0 ), velocity: new Vector2( 0, -1 ) } ),
-      new BodyInfo( { isActive: true, mass: 5, position: new Vector2( 0, -112 ), velocity: new Vector2( 134, 0 ) } ),
-      new BodyInfo( { isActive: true, mass: 5, position: new Vector2( 186, -5 ), velocity: new Vector2( 1, 111 ) } ),
-      new BodyInfo( { isActive: true, mass: 0.000001, position: new Vector2( 70, 72 ), velocity: new Vector2( -47, 63 ) } )
+      new BodyInfo( { isActive: true, mass: 5, position: new Vector2( 0, -1.12 ), velocity: new Vector2( 134, 0 ) } ),
+      new BodyInfo( { isActive: true, mass: 5, position: new Vector2( 1.86, -0.05 ), velocity: new Vector2( 1, 111 ) } ),
+      new BodyInfo( { isActive: true, mass: 0.000001, position: new Vector2( 0.70, 0.72 ), velocity: new Vector2( -47, 63 ) } )
     ] );
     this.orbitalSystemMap.set( OrbitalSystem.BINARY_STAR_PLANET, [
-      new BodyInfo( { isActive: true, mass: 150, position: new Vector2( -100, 0 ), velocity: new Vector2( 0, -60 ) } ),
-      new BodyInfo( { isActive: true, mass: 120, position: new Vector2( 100, 0 ), velocity: new Vector2( 0, 50 ) } ),
-      new BodyInfo( { isActive: true, mass: 0.000001, position: new Vector2( -50, 0 ), velocity: new Vector2( 0, 120 ) } )
+      new BodyInfo( { isActive: true, mass: 150, position: new Vector2( -1.00, 0 ), velocity: new Vector2( 0, -60 ) } ),
+      new BodyInfo( { isActive: true, mass: 120, position: new Vector2( 1.00, 0 ), velocity: new Vector2( 0, 50 ) } ),
+      new BodyInfo( { isActive: true, mass: 0.000001, position: new Vector2( -0.50, 0 ), velocity: new Vector2( 0, 120 ) } )
     ] );
     this.orbitalSystemMap.set( OrbitalSystem.FOUR_STAR_BALLET, [
-      new BodyInfo( { isActive: true, mass: 120, position: new Vector2( -100, 100 ), velocity: new Vector2( -50, -50 ) } ),
-      new BodyInfo( { isActive: true, mass: 120, position: new Vector2( 100, 100 ), velocity: new Vector2( -50, 50 ) } ),
-      new BodyInfo( { isActive: true, mass: 120, position: new Vector2( 100, -100 ), velocity: new Vector2( 50, 50 ) } ),
-      new BodyInfo( { isActive: true, mass: 120, position: new Vector2( -100, -100 ), velocity: new Vector2( 50, -50 ) } )
+      new BodyInfo( { isActive: true, mass: 120, position: new Vector2( -1.00, 1.00 ), velocity: new Vector2( -50, -50 ) } ),
+      new BodyInfo( { isActive: true, mass: 120, position: new Vector2( 1.00, 1.00 ), velocity: new Vector2( -50, 50 ) } ),
+      new BodyInfo( { isActive: true, mass: 120, position: new Vector2( 1.00, -1.00 ), velocity: new Vector2( 50, 50 ) } ),
+      new BodyInfo( { isActive: true, mass: 120, position: new Vector2( -1.00, -1.00 ), velocity: new Vector2( 50, -50 ) } )
     ] );
     this.orbitalSystemMap.set( OrbitalSystem.DOUBLE_DOUBLE, [
-      new BodyInfo( { isActive: true, mass: 60, position: new Vector2( -115, -3 ), velocity: new Vector2( 0, -154 ) } ),
-      new BodyInfo( { isActive: true, mass: 70, position: new Vector2( 102, 0 ), velocity: new Vector2( 1, 150 ) } ),
-      new BodyInfo( { isActive: true, mass: 55, position: new Vector2( -77, -2 ), velocity: new Vector2( -1, 42 ) } ),
-      new BodyInfo( { isActive: true, mass: 62, position: new Vector2( 135, 0 ), velocity: new Vector2( -1, -52 ) } )
+      new BodyInfo( { isActive: true, mass: 60, position: new Vector2( -1.15, -0.03 ), velocity: new Vector2( 0, -154 ) } ),
+      new BodyInfo( { isActive: true, mass: 70, position: new Vector2( 1.02, 0 ), velocity: new Vector2( 1, 150 ) } ),
+      new BodyInfo( { isActive: true, mass: 55, position: new Vector2( -0.77, -0.02 ), velocity: new Vector2( -1, 42 ) } ),
+      new BodyInfo( { isActive: true, mass: 62, position: new Vector2( 1.35, 0 ), velocity: new Vector2( -1, -52 ) } )
     ] );
     this.orbitalSystemMap.set( OrbitalSystem.CUSTOM, [
-      new BodyInfo( { isActive: true, mass: 120, position: new Vector2( -100, 100 ), velocity: new Vector2( -50, -50 ) } ),
-      new BodyInfo( { isActive: true, mass: 120, position: new Vector2( 100, 100 ), velocity: new Vector2( -50, 50 ) } ),
-      new BodyInfo( { isActive: true, mass: 120, position: new Vector2( 100, -100 ), velocity: new Vector2( 50, 50 ) } ),
-      new BodyInfo( { isActive: true, mass: 120, position: new Vector2( -100, -100 ), velocity: new Vector2( 50, -50 ) } )
+      new BodyInfo( { isActive: true, mass: 120, position: new Vector2( -1.00, 1.00 ), velocity: new Vector2( -50, -50 ) } ),
+      new BodyInfo( { isActive: true, mass: 120, position: new Vector2( 1.00, 1.00 ), velocity: new Vector2( -50, 50 ) } ),
+      new BodyInfo( { isActive: true, mass: 120, position: new Vector2( 1.00, -1.00 ), velocity: new Vector2( 50, 50 ) } ),
+      new BodyInfo( { isActive: true, mass: 120, position: new Vector2( -1.00, -1.00 ), velocity: new Vector2( 50, -50 ) } )
     ] );
   }
 }
Index: solar-system-common/js/model/SolarSystemCommonModel.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/solar-system-common/js/model/SolarSystemCommonModel.ts b/solar-system-common/js/model/SolarSystemCommonModel.ts
--- a/solar-system-common/js/model/SolarSystemCommonModel.ts	(revision 8bc3891932b141a5a21ad0ba62170d6c0b24994b)
+++ b/solar-system-common/js/model/SolarSystemCommonModel.ts	(date 1697597966154)
@@ -258,7 +258,7 @@
         // Setting initial values and then resetting the body to make sure the body is in the correct state
         this.bodies[ i ].massProperty.setInitialValue( bodyInfo.mass );
         this.bodies[ i ].positionProperty.setInitialValue( bodyInfo.position );
-        this.bodies[ i ].velocityProperty.setInitialValue( bodyInfo.velocity );
+        this.bodies[ i ].velocityProperty.setInitialValue( bodyInfo.velocity.timesScalar( SolarSystemCommonConstants.POSITION_MULTIPLIER ) );
         this.bodies[ i ].reset();
         if ( preventCollision ) {
           this.bodies[ i ].preventCollision( this.activeBodies );
Index: solar-system-common/js/SolarSystemCommonConstants.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/solar-system-common/js/SolarSystemCommonConstants.ts b/solar-system-common/js/SolarSystemCommonConstants.ts
--- a/solar-system-common/js/SolarSystemCommonConstants.ts	(revision 8bc3891932b141a5a21ad0ba62170d6c0b24994b)
+++ b/solar-system-common/js/SolarSystemCommonConstants.ts	(date 1697597106151)
@@ -63,7 +63,7 @@
   NUMBER_DISPLAY_FONT: new PhetFont( 16 ),
 
   MASS_SLIDER_STEP: 25,
-  GRID_SPACING: 100,
+  GRID_SPACING: 1,
   VBOX_SPACING: 7,
   DEFAULT_SOUND_OUTPUT_LEVEL: 0.2,
 
Index: my-solar-system/js/common/model/MySolarSystemModel.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/my-solar-system/js/common/model/MySolarSystemModel.ts b/my-solar-system/js/common/model/MySolarSystemModel.ts
--- a/my-solar-system/js/common/model/MySolarSystemModel.ts	(revision 1d7455af46b4819f625f0eba5198db0be6e6e513)
+++ b/my-solar-system/js/common/model/MySolarSystemModel.ts	(date 1697597189768)
@@ -46,7 +46,7 @@
     super( options );
 
     this.zoomScaleProperty = new DerivedProperty( [ this.zoomLevelProperty ], zoomLevel => {
-      return Utils.linear( options.zoomLevelRange.min, options.zoomLevelRange.max, 0.25, 1.25, zoomLevel );
+      return Utils.linear( options.zoomLevelRange.min, options.zoomLevelRange.max, 25, 125, zoomLevel );
     } );
 
     this.isLab = options.isLab;

</details>

**Comment by AgustinVallejo** [comment]
Nevermind the last question. I think I'll be able to have this ready by Stand up

**Comment by AgustinVallejo** [comment]
Many changes done, and still more to do. However, I'm pushing before a small break and because both sims are currently working. What's left is polishing corners of the refactoring.

Also, this issue is closely related to this discussion back in april: https://github.com/phetsims/my-solar-system/issues/116. Perhaps we could end up using some of the ideas generated in there.

In the above commits:
- Managed to disregard model units altogether. Instead of using the multipliers, I realized if the value of G changed, we could use the desired units (AU, km/s, and 1e28kg) both in the model and the view, without having to fight huge or tiny numbers. This is consistent with a comment back in april (https://github.com/phetsims/my-solar-system/issues/160) where the user calculated G in the view should be 4.46. I reached the same conclusion and implemented a G value of 4.4567, which is more precise.
  - Not only made sure all the presets are still working correctly, but apparently, with this change, the system became way faster, so I increased the number of iterations in the engine (commit ee68945) and it's now even more accurate. For example, the Sun-Planet-Moon system used to decay and crash after about 3 years in fast mode, now it doesn't de-stabilize at all (see attached figure). Paging in here @arouinfar and @DianaTavares to make sure if this is something we want, as it's possible teachers might have already designed activities around the fact that some of these systems were unstable in the long term.
- Changed all the presets for Kepler, MSS Intro and MSS Lab, transforming positions and velocities into standard units.
- Adjusted the zoom level for most items in the sim, so now instead of 1 it's 100. It's possible there's a more direct way to achieve this effect, maybe changing the screenBounds?
- Added a constant VELOCITY_TO_VIEW_MULTIPLIER because in the past, velocity had about the same scale as position, but now one grid line corresponds to 1AU and also about 23 km/s, depending what you're dragging.

Things that are still broken:
- MeasuringTapeNode hasn't been set to the new scale.
- Anything time related is not working correctly after the change. So stopwatches and the sim clock need revision.
- Unit Tests: Back in April, unit tests were created to make sure we were applying conversions properly, this should get looked into.
- Weird drag bounds: In KeplersLawsScreenView.ts:120 there's the following line:
```ts
point = modelDragBoundsProperty.value.eroded( radius ).closestPointTo( point );
```
Which is not adjusting to the zoom but using the point dragged on the screen. After the units change, this began causing some really weird drag behavior, so it's commented out for now.
- Path Length for MSS
- Areas value for Second Law
- Third Law Graph scale for Third Law
- Cueing Arrows should be scaled.

**Comment by pixelzoom** [comment]
After moving UnitsTests.ts into SolarSystemCommonConstants, this is ready to close.  If any relate problem are identified, we'll open new issues.
